.github/workflows/build.yml

```diff
@@ -26,16 +26,23 @@ jobs:
         os: [ubuntu-latest, macos-latest, windows-latest]
 
     steps:
-      - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+      - uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
```

Comment:
wrong merge , revert changes in this file

Reply:
Revert to what? Please point the exact version, so it's possible to rebase and resolve conflicts with upstream!
Now it is exactly same as actual version
https://github.com/zone117x/MimeMapping/blob/master/.github/workflows/build.yml

Reply:
this change should not be visible here

Reply:
Had no other way to manually fix upstream conflicts, as to take the actual original version and manually merge it into this PR. I can revert this, but this will lead to upstream conflicts.